From 6da049ba9f664478a8139ef78f238d3def3525a2 Mon Sep 17 00:00:00 2001
From: Li Lingfeng <lilingfeng3@huawei.com>
Date: Wed, 19 Oct 2022 12:27:17 +0800
Subject: [PATCH] add CONFIG_BLK_DEV_DUMPINFO and set it enabled in
 openeuler_defconfig
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6da049ba9f664478a8139ef78f238d3def3525a2
Modified-by-SEL: No


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I53Q6M
CVE: NA

--------------------------------

openEuler need detect conflict of opening block device, so
enable it as default.

Signed-off-by: Li Lingfeng <lilingfeng3@huawei.com>
Reviewed-by: Zhang Yi <yi.zhang@huawei.com>
Reviewed-by: Chao Liu <liuchao173@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index ca5f0414b46d..d789a2ca3585 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -910,6 +910,7 @@ CONFIG_BLK_DEBUG_FS=y
 CONFIG_BLK_DEBUG_FS_ZONED=y
 # CONFIG_BLK_SED_OPAL is not set
 # CONFIG_BLK_INLINE_ENCRYPTION is not set
+CONFIG_BLK_DEV_DUMPINFO=y
 
 #
 # Partition Types
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 0009d2ab79e6..228a657e4fa8 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -899,6 +899,7 @@ CONFIG_BLK_DEBUG_FS=y
 CONFIG_BLK_DEBUG_FS_ZONED=y
 # CONFIG_BLK_SED_OPAL is not set
 # CONFIG_BLK_INLINE_ENCRYPTION is not set
+CONFIG_BLK_DEV_DUMPINFO=y
 
 #
 # Partition Types
-- 
2.33.0

